package.json

```diff
@@ -24,7 +24,8 @@
     "frontend"
   ],
   "scripts": {
-    "heroku-postbuild": "git clone https://github.com/mozilla-l10n/fx-private-relay-l10n.git privaterelay/locales; cd frontend; NODE_ENV=\"development\" npm ci; npm run build"
```

Comment:
Huh, I don't recall resolving a conflict here, but looks like Git reverted https://github.com/mozilla/fx-private-relay/pull/1987 when I rebased https://github.com/mozilla/fx-private-relay/pull/1891 :/ I'll chalk that up under the downsides of having PRs open for too long....